src/task.js

```diff
@@ -68,7 +68,7 @@ element.addEventListener('click', (e) => {
       choresArr[i].ID = i + 1;
     }
     localStorage.setItem('choresArr', JSON.stringify(choresArr));
-    // clearChores();
+    clearChores();
```

Comment:
- [ ] I noticed that this function `clearChores()` is not defined and I can't delete any task, Kindly make sure that this function works as expected 👌

![image](https://user-images.githubusercontent.com/45231276/179381550-facab394-8b15-46dc-8b96-2c482a8f6f18.png)